src/cast_core/src/MultiCollectionUpdate.cpp

```diff
@@ -82,8 +83,7 @@ void MultiCollectionUpdate::run() {
 MultiCollectionUpdate::MultiCollectionUpdate(genny::ActorContext& context)
     : Actor(context),
       _rng{context.workload().createRNG()},
-      _updateTimer{context.timer("updateTime", MultiCollectionUpdate::id())},
-      _updateCount{context.counter("updatedDocuments", MultiCollectionUpdate::id())},
+      _updateOp{context.operation("updateOperation", MultiCollectionUpdate::id())},
```

Comment:
no need for the "operation" suffix and let's use PascalCase. So just call it `context.operation("Update")`

Reply:
Currently all of the actors use camelCase in these situations. Do we prefer if they were all PascalCase?

Reply:
> Do we prefer if they were all PascalCase?

Yes.

Reply:
👍